infra/cray_infra/one_server/create_vllm.py

```diff
@@ -11,6 +11,15 @@
 
 logger = logging.getLogger(__name__)
 
+def _is_cuda() -> bool:
+    has_cuda = torch.version.cuda is not None
+    return VLLM_TARGET_DEVICE == "cuda" and has_cuda and not (_is_neuron() or _is_tpu())
+
+
+def _is_hip() -> bool:
+    return (
+        VLLM_TARGET_DEVICE == "cuda" or VLLM_TARGET_DEVICE == "rocm"
+    ) and torch.version.hip is not None
```

Comment:
I am unsure about the appropriate place for this call... keeping it here for now.